File: app/src/main/java/com/adityap/flashy_createflashcards/PlayCardActivity.java
package com.adityap.flashy_createflashcards;

import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.adityap.flashy_createflashcards.adapters.CustomPagerAdapter;
import com.adityap.flashy_createflashcards.models.DeckModel;

public class PlayCardActivity extends AppCompatActivity {

    FlashcardDatabaseHelper mFlashcardDatabaseHelper;
    ViewPager mViewPager;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_play_card);

        mFlashcardDatabaseHelper = new FlashcardDatabaseHelper(this);
        mFlashcardDatabaseHelper.getReadableDatabase();

        Bundle bundle;
        bundle = getIntent().getExtras();
        final DeckModel deckModel = bundle.getParcelable("Deck");

        mViewPager = findViewById(R.id.viewPager);
        mViewPager.setAdapter(new CustomPagerAdapter(this,mFlashcardDatabaseHelper.readFlashcards(deckModel.getId())));
    }
}

File: app/src/main/java/com/adityap/flashy_createflashcards/PlayCardActivity.kt
package com.adityap.flashy_createflashcards

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.adityap.flashy_createflashcards.adapters.CardAdapter
import com.adityap.flashy_createflashcards.database.DatabaseHelper
import com.adityap.flashy_createflashcards.database.DatabaseHelperFactory
import com.yuyakaido.android.cardstackview.CardStackLayoutManager
import kotlinx.android.synthetic.main.activity_play_card.*

class PlayCardActivity : AppCompatActivity() {
    lateinit var databaseHelper: DatabaseHelper

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_play_card)

        supportActionBar?.setDisplayHomeAsUpEnabled(true)


        databaseHelper = DatabaseHelperFactory.getDBHelper(this)
        val bundle: Bundle? = intent.extras
        val deckModelId: Int? = bundle?.getInt("Deck")
        val listOfCards = databaseHelper.readFlashcards(deckModelId)

        card_stack_view.layoutManager = CardStackLayoutManager(baseContext);
        card_stack_view.adapter = CardAdapter(listOfCards)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // handle arrow click here
        if (item.itemId === android.R.id.home) {
            finish() // close this activity and return to preview activity (if there is any)
        }
        return super.onOptionsItemSelected(item)
    }
}
